Code/Editor/EditorPreferencesPageViewportGeneral.cpp

```diff
@@ -46,10 +48,7 @@ void CEditorPreferencesPage_ViewportGeneral::Reflect(AZ::SerializeContext& seria
         ->Field("ShowGridGuide", &Display::m_showGridGuide)
         ->Field("DisplayDimensions", &Display::m_displayDimension);
 
-    serialize.Class<MapViewport>()
-        ->Version(1)
-        ->Field("SwapXY", &MapViewport::m_swapXY)
-        ->Field("Resolution", &MapViewport::m_resolution);
+    serialize.Class<MapViewport>()->Version(1)->Field("SwapXY", &MapViewport::m_swapXY)->Field("Resolution", &MapViewport::m_resolution);
```

Comment:
this was way more readable before, can we keep it like it was?

Reply:
I'll put a `clang-format off/on` block around it and revert the styling (I do think autoformatting is just so much simpler for nearly everything though, just accepting _this is the style_ does make life a lot simpler, but I can understand in this context it sometimes helping

Reply:
Now updated :+1: